dummyfs/dummyfs.c

```diff
@@ -151,6 +149,8 @@ int dummyfs_lookup(void *ctx, oid_t *dir, const char *name, oid_t *res, oid_t *d
 		d = object_get(fs, res->id);
 	}
 
+	object_put(fs, d);
```

Comment:
If founded object is a device, then for one object_get, two object_put will be called (one after sucesscul dir_find(), and second after exiting loop)

Reply:
Right, fixed, thanks!